UnityProject/Assets/Scripts/HealthV2/Living/Mutations/BigStomachMutation.cs

```diff
@@ -0,0 +1,35 @@
+using System.Collections;
+using System.Collections.Generic;
+using HealthV2;
+using UnityEngine;
+
+[CreateAssetMenu(fileName = "BigStomachMutation", menuName = "ScriptableObjects/Mutations/BigStomachMutation")]
+public class BigStomachMutation : MutationSO
+{
+
+	public override Mutation GetMutation(BodyPart BodyPart,MutationSO _RelatedMutationSO)
+	{
+		return new InBigStomachMutation(BodyPart,_RelatedMutationSO);
+	}
+
+	public class InBigStomachMutation : Mutation
+	{
+		public InBigStomachMutation(BodyPart BodyPart,MutationSO _RelatedMutationSO) : base(BodyPart,_RelatedMutationSO)
```

Comment:
wot? what does this or do what is its use?

Reply:
basically so forces you to use the constructor and pass in the variables

Reply:
ah, good then.